deploy/deploy-with-olm.yaml

```diff
@@ -17,7 +17,7 @@ spec:
     metadata:
       creationTimestamp: null
   targetNamespaces:
-  - openshift-storage
+    openshift-storage
```

Comment:
Why this hunk in the commit "Added the redhat logo on the operator hub" ?

Reply:
This is even wrong and lets the test fail, I think:

```
uncommitted deploy/deploy-with-olm.yaml changes. run 'make gen-latest-deploy-yaml' and commit results.
make: *** [Makefile:94: verify-latest-deploy-yaml] Error 1
```

Reply:
I'll fix this in a moment.